libs/feedback/src/lib/feedback-widget/feedback-widget.component.html

```diff
@@ -14,56 +14,76 @@ <h2 class="modal-title">Feedback</h2>
       <ng-container *ngIf="(messages$ | async) as message">
         <ng-container *ngIf="message.length">
           <div class="feedback-items">
-            <div class="feedback-item" *ngFor="let item of (messages$ | async)">
-              <div class="header">
-                <span class="name">{{ item.name }}</span>
-                <span class="date"> {{ item.timestamp | date }}</span>
+            <ng-container *ngFor="let item of (messages$ | async)">
+              <div class="feedback-item">
+                <div class="dropdown">
+                  <button class="dropdown-btn">
+                    <img alt="" src="/assets/images/icons/icon-tools.png"/>
+                  </button>
+                  <div class="dropdown-content">
+                    <a (click)="createAnIssue(item)"
+                       class="dropdown-content-item">
+                      Take
+                    </a>
+                    <a (click)="createClosedIssue(item, '[Duplicate]')"
+                       class="dropdown-content-item">
+                      Close [Duplicate] </a>
+                    <a (click)="createClosedIssue(item, '[No fix]')"
+                       class="dropdown-content-item">
+                      Close [No fix]
+                    </a>
+                    <a (click)="createClosedIssue(item, '[Done]')"
+                       class="dropdown-content-item">
+                      Close [Done]
+                    </a>
+                    <a (click)="createClosedIssue(item, '[Nice message, though not a real bug]')"
+                       class="dropdown-content-item">
+                      Close [Nice message]
+                    </a>
+                    <a (click)="createClosedIssue(item, '[Can\'t repcoduce]')"
+                       class="dropdown-content-item">
+                      Close [Can't reproduce]
+                    </a>
```

Comment:
Seems like it could be a separate component

Reply:
It depends on the business logic, for now for me it was just a button with method)) And it's used as it was in feedback-page component

On the next step I'm gonna get how we want it to work and make separate components